jc/parsers/metamf.py

```diff
@@ -0,0 +1,228 @@
+"""jc - JSON CLI output utility `metamf` command output parser
+
+Options supported:
+- none
+
+Create key value pairs from a MANIFEST.MF file, to include key multiline value pairs.
```

Comment:
Don't need these lines in the docstring for this parser. Since it is a filetype parser, there are no command options to support. The intent of the parser should be clear by the name and metadata.